src/test/java/com/jcabi/github/mock/MkIssueTest.java

```diff
@@ -253,6 +255,53 @@ public void assignsUser() throws Exception {
     }
 
     /**
+     * MkIssue creates a closed event when closing an issue.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void closingGeneratesEvent() throws Exception {
```

Comment:
@cvrebert let's rename the test to something like `createsClosedEvent`, the name should start with a verb.